website/static/js/koHelpers.js

```diff
@@ -259,6 +260,15 @@ ko.bindingHandlers.tooltip = {
     init: tooltip,
     update: tooltip
 };
+
+var clipboard = function(el, valueAccessor) {
+    makeClient($(el));
```

Comment:
Minor: `makeClient` expects to receive a DOM element, not a jQuery object, so no need to do `$()`. It should be handled fine, though.